mirrulations-client/tests/test_client.py

```diff
@@ -211,6 +211,33 @@ def test_client_returns_403_error_to_server(mock_requests):
         assert '403' in response.json()
 
 
+def test_mock_get_timeout(mock_requests):
+    server_validator = Validator('http://test.com/')
+
+    with mock_requests:
+        mock_requests.get('http://test.com/http://test.com/get_results', exc=requests.exceptions.Timeout)
+        try:
+            server_validator.get_request('http://test.com/get_results')
+        except requests.exceptions.Timeout as error:
```

Comment:
The test should fail if the system does *not* throw the exception.  You should call `pytest.fail()` as a second line of the `try` block.